### PR TITLE
Add landing page for Rust workshop

### PR DESCRIPTION
**Pull request by marcoow**
see design here: https://www.sketch.com/s/7baa7317-7ec9-48b5-9584-4edb6163c143/a/3oo5EVm

Preview URL: https://deploy-preview-2038--objective-northcutt-37494c.netlify.app/web-based-services-in-rust-workshop/

This page isn't linked from anywhere on mainmatter.com – it will be served via https://rust-web-services-workshop.mainmatter.com

## TODO

- [ ] add `og:image`

localhost_8080_web-based-services-in-rust-workshop_

### CODE REVIEW

**Comment by davidhajba** [comment]
Everything looks great on my side, except one thing:

* I still don't think it is a good idea using parallax scrolling with the purple overlay in the Book your ticket section.
* On desktop the image without the overlay looks bad quality photo, because it's optimized to work together perfectly with the overlay
* But on mobile it's a serious UX problem, because the content of this section is not readable. As this is the most important part of the page, where the user can buy ticket, imo it is not acceptable to have any readability problem here.
* I know this is a general thing around all Mainmatter pages, I would suggest fixing this asap

Screenshot 2023-05-16 at 09 11 37

Screenshot 2023-05-16 at 09 10 15

**Review by oscard0m — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -141,6 +141,12 @@
   from = "/why-simplabs"
   to = "/"
 
+# serve "Web-based Service in Rust" landing page for rust-web-services-workshop.mainmatter.com
```
Comment (SaeLLo) [inline netlify.toml]: It has to be plural not singular -> Services

File: src/web-based-services-in-rust-workshop.njk [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+            <li class="lang-features__list-item">
+              <p class="lang-features__feature h4">
+                {% include 'svg/arrow.njk' %}
+                <span class="lang-features__feature-text">Library availability</span>
+              </p>
+            </li>
+            <li class="lang-features__list-item">
+              <p class="lang-features__feature h4">
+                {% include 'svg/arrow.njk' %}
+                <span class="lang-features__feature-text">Concurrency approach</span>
```
Comment (SaeLLo) [inline src/web-based-services-in-rust-workshop.njk]: We have "Concurrecny Approach" twice

File: src/web-based-services-in-rust-workshop.njk [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+              </p>
+            </li>
+          </ul>
+        </div>
+        <p>Developing web applications and web services. As a result, you can create services that are fast, reliable, and cost-effective. In fact, using Rust to write services can tremendously reduce your operating costs. However, the main challenge is knowing where to start. This workshop aims to guide you through the process of building web services using the ecosystem around the Tokio async runtime. You will learn how to make basic API calls, create advanced middleware layers to add functionality, and conduct tracing and testing. Join us for three afternoons to acquire all the necessary skills to produce your own Rust-based web apps.</p>
```
Comment (SaeLLo) [inline src/web-based-services-in-rust-workshop.njk]: ...for developing...
This is the second part of the sentence before

File: src/web-based-services-in-rust-workshop.njk [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+          "loading": "lazy",
+          "sizesArray": [760, 1440, 1920]
+        } %}
+        {{ imageAspectRatio(imageData, "32/13", "35/19") }}
+      </section>
+
+      <section class="lp-section">
+        <div class="container container--lg">
+          <h2 class="lp-section-subheading h2">Mode</h2>
+          <p>The workshop will be held over three afternoons, and each session will include practical examples. In addition, we will provide optional homework for you to work on. For who? This workshop is intended for developers who have some experience with Rust and want to learn more about its problem domain for web applications. It is not necessary to have written Rust in a production environment yet. If you are just starting out with Rust, we will provide you with preparatory material beforehand.</p>
```
Comment (SaeLLo) [inline src/web-based-services-in-rust-workshop.njk]: Delete "For who? This workshop is intended for developers who have some experience with Rust and want to learn more about its problem domain for web applications. It is not necessary to have written Rust in a production environment yet. If you are just starting out with Rust, we will provide you with preparatory material beforehand"

**Comment by marcoow** [comment]
@davidhajba we cannot disable the overlay animation easily in the footer? Is there a scenario in which the overlay does not cover the background photo so that you can't read the text and you can't scroll in a way that the overlay correctly covers the image? I can't reproduce that…

**Comment by davidhajba** [comment]
> we cannot disable the overlay animation easily in the footer?
* This would be the best to disable the overlay I would say.

> overlay does not cover the background photo so that you can't read the text and you can't scroll in a way that the overlay correctly covers the image

* As I see on mobile this is the case check [HERE](https://www.loom.com/share/837e8ab8789b46609eade10f45bc24f9) 
* I don't think this should be a challenge for the User - try to scroll the screen in the right position to be able to read the content. Content should be clearly readable all the time
* I would not suggest using any decoration (like parallax scrolling) which is against readability

**Comment by marcoow** [comment]
@davidhajba figured out a way to disable the animation. Please can you create an `og:image` for this?

**Comment by davidhajba** [comment]
Great, I think it is much better without the animation - thank you.

I'm happy to create an OG Image - what should be the content?

**Review by davidhajba — APPROVED** [review]

**Comment by marcoow** [comment]
see the `og:image`s of the other workshops for reference

**Comment by davidhajba** [comment]
I see - [HERE](https://www.sketch.com/s/066e539c-c206-4261-aafb-9634ab577579/a/5nxJWJy#Version) you are

**Pull request merged**